doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -81,6 +81,7 @@ Doc. ID: UG-SYSPAR
 | 30.09.2022 | 2.68    | Updated *key-length* property EE- and FI-package values.                                                                                                                                                                                                                                                                                                                                                       | Petteri Kivimäki           |
 | 30.01.2023 | 2.69    | Updated [13.3 Automatic Backups](#133-automatic-backups) chapter with latest configuration additions.                                                                                                                                                                                                                                                                                                          | Ričardas Bučiūnas          |
 | 26.05.2023 | 2.70    | Add rate-limit properties                                                                                                                                                                                                                                                                                                                                                                                      | Ričardas Bučiūnas          |
+| 01.06.2023 | 2.71    | Update and add missing descriptions for to Central server configuration parameters.                                                                                                                                                                                                                                                                                                                            | Ovidijus Narkevičius       |
```

Comment:
The version number in the beginning of the document should be updated from 2.70 to 2.71.